### PR TITLE
Configurable redirect

### PR DESCRIPTION
**Pull request by derrickp**
We can now configure where we'd like to send users after signup. I'm not 100% sure I love the name of `redirect.success` so if you have another suggestion I'd love to hear it. But either way, the functionality is there.

Completes https://github.com/minimail/minimail/issues/19

### CODE REVIEW

**Comment by jgayfer** [comment]
What about "subscription redirect" or "subscribed redirect"?

Something to indicate that the action is subscribing, and we're setting the resulting URL from that action.

File: README.md [hunk 1]
```diff
@@ -20,3 +20,13 @@ cargo sql migrate add <name_of_migration>
 ### Authenticated Requests
 
 The endpoint for retrieving subscribers is locked behind an admin token that is set through the env. Set the env variable `ADMIN_TOKEN` to whatever value you want and just ensure you pass it whenever you are accessing the endpoint inside of the `Authorization` header, using `Bearer <ADMIN_TOKEN>`.
+
+### Configurable Redirect Location
+
+If you would like to configure where `minimail` should send users upon a successful save, the address can be filled in the configuration file by adding something like the following.
```
Comment (jgayfer) [inline README.md:26]: Maybe add a note about origin being the default?

**Review by jgayfer — APPROVED** [review]
Awesome! I'll have to set this up soon.

File: README.md (hunk 1 above)
Comment (derrickp) [reply README.md:26]: Good idea. I’ll add that

**Comment by derrickp** [comment]
> What about "subscription redirect" or "subscribed redirect"?
> 
> Something to indicate that the action is subscribing, and we're setting the resulting URL from that action.

@jgayfer  What if I called it redirect -> subscribed. That way if we ever had other redirects they could go in there, like if we wanted an error or unsubscribed one.

**Pull request merged**